test/general/test_numerics.cpp

```diff
@@ -367,6 +368,42 @@ TEST(ctmath, sum)
     EXPECT_DOUBLE_EQ(dfunctor->eval(.5), omega * (cos(omega * .5) - sin(omega * .5)));
 }
 
+TEST(ctmath, new_sum)
+{
+    auto functor = newSumFunction(newFunc1("sin"), newFunc1("sin"));
+    EXPECT_EQ(functor->type(), "times-constant");
+    EXPECT_DOUBLE_EQ(functor->eval(.5), 2 * sin(.5));
```

Comment:
Should 
```suggestion
    double x = .5;
    EXPECT_DOUBLE_EQ(functor->eval(x), 2 * sin(x));
```
Should use variable for clarity (here and on most other additions in this file).